recipe/meta.yaml

```diff
@@ -28,11 +27,16 @@ requirements:
     - pkg-config  # [not win]
     - make  # [not win]
     - m2-patch  # [win]
+    - python >=3.10,<3.11  # [not win]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - libiconv  # [not win]
   host:
     - icu  # [not win]
     - libiconv
     - xz  # [not win]
     - zlib
+    - python >=3.10,<3.11  # [not win]
+    - libiconv  # [not win]
```

Comment:
I'm not sure this is the best solution, probably not, but it passes on my machine. Let's see how it goes here.

Reply:
Thanks @ocefpaf !

Reply:
PR to remove python from host requirements #99